### PR TITLE
Take 2 to fix SDP version stuff

### LINKED ISSUE #239: When is the SDP sess-version in o= line updated
When the SDP changes, including new trickle ICE candates, the sess-version needs to go up. Need to decide how to do this and add text to support. 

See http://tools.ietf.org/html/rfc4566#section-5.1

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #239

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2344,9 +2344,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                        increment is a call to the createOffer()
+                        API. If the Javascript has provided an offer
+                        to a remote side that might be confused by
+                        receiving different SDP with the same
+                        &lt;session-version&gt;, it should ensure that
+                        the createOffer() API is used to create the
+                        SDP such that all &lt;session-version&gt; of
+                        the SDP that the remote side will see have a
+                        unique &lt;session-version&gt;.</t>
+
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Regrettably, this is not what we agreed:

"V is the version in the o=line in last call to setLocal
o=V+1 on CreateOffer() if if anything changes 
"

**Comment by ekr** [comment]
From meeting minutes
"SDP o= line increment [#239]
While the proposed solution on the slide was accepted, it was noted that the proposal on the slide goes against what was agreed at last meeting. Emil Ivov also commented that Trickle ICE for SIP does exactly what is described on the slide. Jonathan Lennox said that we should make sure this is compatible with whatever is done in trickle ICE SIP stacks."

**Comment by ekr** [comment]
This algorithm becomes correct once we decide to not allow o= modification. LGTM

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -2344,9 +2344,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>The fields of the "o=" line MUST stay the
                         same except for the &lt;session-version&gt;
-                        field, which MUST increment if the session
-                        description changes in any way, including the
-                        addition of ICE candidates.</t>
+                        field, which MUST increment for each call to
+                        the createOffer() API. The only actions which
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: I think actually:
Which MUST be incremented on each call to the createOffer method which would result in a different offer from the current local description. Implementations MAY
opt to increment the "o=" line on every call to createOffer.

**Comment by juberti** [comment]
Discussion in-room:
1) add example to make it clear that there is a separate counter (other than current o=) which is what increments.
2) change text about &lt;session-version&gt; to be more of a "note that" for an edge case, as opposed to normative text about a mainline case.

**Comment by juberti** [comment]
Updated this branch. PTAL

**Comment by fluffy** [comment]
LGTM

**Comment by ekr** [comment]
2349: "might differ" is creepy. "differs" perhaps.

2359: "N + 2"  -> must be greater than N + 1

This is basically OK modulo the above

**Pull request merged**